desktop/desktop.scss

```
@@ -51,26 +51,70 @@
 }
 
 // Recent replies
-.recent-replies--reply {
-  border-bottom: 1px solid var(--primary-low);
-  padding-bottom: 0.5em;
-  margin-bottom: 1em;
-  display: flex;
-}
+.recent-replies {
+  &--reply {
+    display: flex;
+    &:not(:last-child) {
+      padding-bottom: 1em;
+      border-bottom: 1px solid var(--primary-low);
+      margin-bottom: 1em;
+    }
+  }
 
-.recent-replies--col:first-child {
-  padding-top: 0.05em;
-}
-.recent-replies--col:nth-child(2) {
-  margin: 0 0.5em;
-  flex-shrink: 1;
-  max-width: 80%;
-  word-break: break-word;
+  &--col {
+    &:first-child {
+      padding-top: 0.05em;
+    }
+    &:nth-child(2) {
+      margin: 0 0.5em;
+      flex-shrink: 1;
+      max-width: 80%;
+      word-break: break-word;
+    }
+  }
+
+  &--date {
+    font-size: var(--font-down-1);
+    color: var(--primary-medium);
+  }
 }
 
-.recent-replies--date {
-  font-size: var(--font-down-1);
-  color: var(--primary-medium);
+// top topics, tag topics
+
+.tag-topics,
+.top-topics {
+  &--topic {
```

Could you use BEM according to our guidelines when writing new CSS?

```suggestion
  &__topic {
```

---

I can but I was just matching the existing style in the SCSS, I'll have to update all the existing handlebar files to match

---

I fear changing the existing CSS names will undo a lot of customizations communities may already have in place. Like `recent-replies--reply` to `recent-replies__reply`. Should I just have it done for the ones I added for now even if it won't match the existing style?

---

Good call, let's keep existing classes unchanged.

---

Sorry but I disagree. I'm not asking to change existing classes, but if we go the route of "keep the same style as the old one even for new stuff" we'll never get BEM implemented. 
So I rather use BEM for any new stuff, even if it is a style change from the rest of the file. If we ever get to refactoring, that's less work left.

---

Looks to me like we're in agreement. Using underscores for the new elements is fine with me, I'd rather not edit classes for the existing components in this PR.